source/docs/blockchain_privatenet_configuration.md

```diff
@@ -0,0 +1,43 @@
+title: Privatenet Blockchain configuration
+---
+
+A private network is really similar to using the development mode of a client.
+
+The biggest differences is that it does not come with a default pre-funded account and it will not use POA (proof of authority), meaning that blocks will need to be mined.
+
+Fear not, Embark has settings to limit the mining to a minimum so that everything can run smoothly while testing in a more realistic environment before going to a test network.
+
+## Testnet parameters
+
+<pre><code class="javascript">privatenet: {
+  networkType: "custom",
+  networkId: 1337,
+  isDev: false,
+  datadir: ".embark/privatenet/datadir",
+  mineWhenNeeded: true, 
+  genesisBlock: "config/privatenet/genesis.json",
+  nodiscover: true,
+  maxpeers: 0,
+  proxy: true,
+  accounts: [
+    {
+      nodeAccounts: true,
+      password: "config/privatenet/password"
+    }
+  ]
+}
+</code></pre>
+
+Please note, you can always use the parameters we saw in the [previous section](/docs/blockchain_basic_configuration.html) to override the `default` parameters.
+
+### Parameter descriptions
+
+- **isDev**: You need to set `isDev` to false to use enable any network that isn't the development one
```

Comment:
As mentioned in the other comment, it'd be great to have a section to link to that explains when one wants to use `isDev` and what it means.